.env.example

```diff
@@ -1,20 +1,22 @@
 # Base Path Openshift: /businesses/create Firebase: /
 VUE_APP_PATH="/"
 
-#vaults Shared
+# vaults Shared
 VUE_APP_ADDRESS_COMPLETE_KEY=
 
-#vaults web-url
+# vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_DASHBOARD_URL="https://dev.business.bcregistry.gov.bc.ca/"
 VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
 
-#vaults API
+# vaults API
 VUE_APP_AUTH_API_URL="https://auth-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_AUTH_API_VERSION="/api/v1"
-VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
+# for Legal Name feature branch only:
+VUE_APP_LEGAL_API_URL="https://business-api-dy4loprnwa-nn.a.run.app"
+#VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
```

Comment:
This will need to be updated when merging to main branch.

Reply:
@argush3 ^^

... Whatever we end up doing for the new Legal (Name) API URL ...